.storybook/main.js

```diff
@@ -31,6 +32,15 @@ module.exports = {
       }
     });
 
+    const postCssConfig = styles.getPostCssConfig({
+      themeImporter: {
+        themePath: require.resolve('@ckeditor/ckeditor5-theme-lark')
+      },
+      minify: true
+    });
+
+    postCssConfig.plugins.push(...plugins);
```

Comment:
This customization is only required for our local storybook environment. Consumers wouldn't have to worry about this step.